Gems/Atom/RHI/Vulkan/Code/Source/RHI/Buffer.h

```diff
@@ -68,7 +68,7 @@ namespace AZ
 
         private:
             Buffer() = default;
-            RHI::ResultCode Init(Device& device, const RHI::BufferDescriptor& bufferDescriptor, BufferMemoryView& memmoryView);
+            RHI::ResultCode Init(Device& device, const RHI::BufferDescriptor& bufferDescriptor, const BufferMemoryView& memmoryView);
```

Comment:
repeat: The memmoryView variable has a typo in it.
```suggestion
            RHI::ResultCode Init(Device& device, const RHI::BufferDescriptor& bufferDescriptor, const BufferMemoryView& memoryView);
```